pkg/serverless/appsec/httpsec/proxy.go

```diff
@@ -95,7 +97,16 @@ func (lp *ProxyLifecycleProcessor) OnInvokeEnd(_ *invocationlifecycle.Invocation
 }
 
 func (lp *ProxyLifecycleProcessor) spanModifier(lastReqId string, chunk *pb.TraceChunk, s *pb.Span) {
-	// Add appsec tags to the aws lambda function service entry span
+	// Add relevant standalone tags to the chunk (TODO: remove per span tagging once backend handles chunk tags)
+	if isStandalone() {
+		if chunk.Tags == nil {
+			chunk.Tags = make(map[string]string)
+		}
+		chunk.Tags["_dd.apm.enabled"] = "0"
+		for _, s := range chunk.Spans {
```

Comment:
Worth adding a comment saying that the specification says service entry span only but we are putting them everywhere here for extra care (in case the top-level span condition isn't properly met at this point of the agent (because the agent can automatically change a span into a service entry span at some point if the service name is different or if there is no parent span, but I don't know where compared to this function call)).